theme/theme.inc

```diff
@@ -53,13 +53,20 @@ function islandora_preprocess_islandora_default_edit(array &$variables) {
       'class' => 'datastream-size',
       'data' => islandora_datastream_get_human_readable_size($ds),
     );
+    if (user_access(ISLANDORA_VIEW_DATASTREAM_HISTORY)) {
```

Comment:
This is redundant as `islandora_datastream_access` checks `user_access` https://github.com/Islandora/islandora/blob/7.x/islandora.module#L1951